### ISSUE TITLE
Natural satellite (Portions of atmosphere, cryosphere, hydrosphere, lithosphere)

### ISSUE THREAD

**Issue by alanruttenberg** [CommonCoreOntology/CommonCoreOntologies#249]
All use "natural satellite" in their definitions, but natural satellite isn't defined. I've been using the terms "astronomical body" defined as: A material entity that is a natural resident space object that is a single, cohesive structure bound together by gravity and/or electromagnetism.

"Natural resident space object" is adapted from https://en.wikipedia.org/wiki/Resident_Space_Object

"Satellite" connotes either the artifact, or a moon - something in relation to a planet, so isn't as general.

**Comment by alanruttenberg** [comment]
Also, by the definition of Portion of Geosphere, it would seem that Portions of Hydro, Litho, Atmosphere are subclasses of Portion of Geosphere. From the definition of Portion of Hydrosphere it would seem Portion of Cryosphere is both a subclass and part of it?

**Comment by alanruttenberg** [comment]
Finally, along these lines, I need a term that would be something like the material in a site surrounding a material entity, on which climate can depend on. Wind speed, air temperature, water temperature around a ship would be an example of something I need to represent.

**Comment by gregfowlerphd** [comment]
Alan's second point (re: apparent subclass relations) seems correct to me. It's an issue I was also planning to raise.

**Comment by cameronmore** [comment]
@alanruttenberg does 'resident space object' have a connotation of depending on rotating around another space object? Do we want to remove all connotations of 'satellite' and having to rotate around other bodies?

If yes to both, we can opt for changing 'natural satellite' in these definitions to 'astronomical body' and define astronomical body as:

Astronomical body = a material entity that is a naturally-occurring, single, cohesive structure bound together by gravity and/or electromagnetism in the vacuum of space.

**Comment by alanruttenberg** [comment]
Reading the wikipedia page it seems that resident space object assumes orbit around something. By my reading a satellite is a resident space object that is an artifact. So my definition of astronomical body isn't good, since I intend it to include isolated stars, for example.
Re astronomical body, I'm not remembering why I included bound by electromagnetism in the definition.
"Bound together by gravity" works as does "in the vacuum of space"

**Comment by cameronmore** [comment]
In sum, change references of 'natural satellite' to 'astronomical body' and add astronomical body as:

"Astronomical body = a material entity that is a naturally-occurring, single, cohesive structure bound together by gravity in the vacuum of space."

example of usage, "a star, a planet, a comet, a nebula."

source, "derived from Jeanne Hopkins, Glossary of Astronomy and Astrophysics, Chicago, University of Chicago Press, 1982, and a list available here : https://en.wikipedia.org/w/index.php?title=Category:Lists_of_astronomical_objects&oldid=958362983"

**Comment by alanruttenberg** [comment]
Wikipedia makes the distinction between astronomical bodies, which are compact, vs astronomical objects, which include astronomical bodies but also things like nebulae and galaxies. We tend to orbit the bodies, so it might be worth keeping the distinction wikipedia makes.
>An astronomical object, celestial object, stellar object or heavenly body is a naturally occurring [physical entity](https://en.wikipedia.org/wiki/Physical_object), association, or structure that exists within the [observable universe](https://en.wikipedia.org/wiki/Observable_universe).[[1]](https://en.wikipedia.org/wiki/Astronomical_object#cite_note-1) In [astronomy](https://en.wikipedia.org/wiki/Astronomy), the terms object and body are often used interchangeably. However, an astronomical body or celestial body is a single, tightly bound, contiguous entity, while an astronomical or celestial object is a complex, less cohesively bound structure, which may consist of multiple bodies or even other objects with substructures.

**Comment by cameronmore** [comment]
So more like:

`Astronomical body = a material entity that is a naturally-occurring, single, cohesive structure bound together by gravity in the vacuum of space.`
example of usage, "a star, a planet, a comet, a nebula."
source, "derived from Jeanne Hopkins, Glossary of Astronomy and Astrophysics, Chicago, University of Chicago Press, 1982, and a list available here : https://en.wikipedia.org/w/index.php?title=Category:Lists_of_astronomical_objects&oldid=958362983"

`Astronomical object = A material entity that is a naturally occurring, identifiable, structure in the vacuum of space.`
rdfs:comment , an astronomical object may be an aggregate of other, smaller material entities, gases, or hard compact celestial bodies.
source, derived from https://en.wikipedia.org/w/index.php?title=Category:Lists_of_astronomical_objects&oldid=958362983

**Comment by alanruttenberg** [comment]
I don't think nebula counts as astronomical body by this definition. Seems more like astronomical object. But I'm not an astronomer.

**Comment by cameronmore** [comment]
Right, I adjusted the comment. I like this distinction, but I am having a hard time finding some good sources that aren't Wikipedia.

**Comment by alanruttenberg** [comment]
Their citation to https://www.iau.org/public/themes/naming/ is useful. It defines object but in the text you can see how they use the term body.

**Comment by cameronmore** [comment]
I did see that, though ideally there's a glossary we can cite. I was looking through the IAU documentation and website but didn't find anything that fit the bill, except this: https://astro4edu.org/resources/glossary/term/49/ which is a space public education platform, but I think it works.

**Comment by alanruttenberg** [comment]
yes that looks good.

**Comment by APCox** [comment]
Many of the terms (and more) discussed above are already in a domain ontology. Specifically 'Natural Satellite' and its parent class 'Astronomical Body'. This is an unfortunate case of terms in the mid-level relying on terms defined in a domain-level ontology.

**Comment by APCox** [comment]
> Also, by the definition of Portion of Geosphere, it would seem that Portions of Hydro, Litho, Atmosphere are subclasses of Portion of Geosphere. From the definition of Portion of Hydrosphere it would seem Portion of Cryosphere is both a subclass and part of it?

'Portion of Geosphere' is defined as 
`A fiat object part that is composed of one or more portions of the atmosphere, cryosphere, hydrosphere, or lithosphere`

I believe the intent was to provide a term for the collective whole of the separate portions as well as any subsets thereof. I agree that the current wording is logically consistent with each of the other portions being subtypes of 'Portion of Geosphere', though the intention seemed to be to capture a relationship of parthood rather than subclass of. Of course, these parthood relations have not been asserted.

My primary complaint about the above definition is that its label is Earth-specific but the definition tries to make it more generic. I suggest changing its definition to:
`A fiat object part that is composed of one or more portions of Earth's atmosphere, cryosphere, hydrosphere, or lithosphere.`

Since the other portions are not Earth-specific, they would not be subtypes of geosphere.

Note that, if this solution is implemented, 'Material Territory of a Government Domain' should no longer be asserted as a subtype of 'Portion of Geosphere' since it is possible to have a government whose domain includes territory not on Earth.

I agree that 'Portion of Cryosphere' should be a subtype of 'Portion of Hydrosphere'. Also, 'hydrosphere' is misspelled in the definition of cryosphere.
